code/ZAS/Fire.dm

```diff
@@ -388,7 +388,15 @@ datum/gas_mixture/proc/check_recombustability(list/fuel_objs)
 
 	if(total_combustables > 0)
 		//slows down the burning when the concentration of the reactants is low
-		var/damping_multiplier = min(1, active_combustables / (total_moles/group_multiplier))
+		var/damping_multiplier
+		if(!total_moles || !group_multiplier)
+			damping_multiplier = min(1, active_combustables)
+		else if(!total_moles)
+			damping_multiplier = min(1, active_combustables / group_multiplier)
+		else if(!group_multiplier)
+			damping_multiplier = min(1, active_combustables / total_moles)
```

Comment:
total_moles can only be non-zero, right?

Reply:
idk, but it runtimes with division_by 0

Reply:
Wouldn't it still runtime since the division still takes place or am I misunderstanding how byond treats division by zero errors?

Reply:
Not it would not, that is what checks are for
```
else if(!total_moles)
    //something
else if(!group_multiplier)
   // divide by total moles because it is definitely not 0
```

Reply:
Oh I see it. The check confused me. Thank you.